privval/internal/ip_filter.go

```diff
@@ -4,16 +4,17 @@ import (
 	"fmt"
 	"github.com/Finschia/ostracon/libs/log"
 	"net"
+	"strings"
 )
 
 type IpFilter struct {
-	allowAddr string
+	allowList []string
```

Comment:
It's not so large set and is used infrequently, it can be approved as a slice, but using Set (`map[string]struct{}` and `_, ok := allowList[hostAddr]; ok` in golang) is better because it's $O(1)$ in complexity.

Reply:
As you may expect, this is not so large and is used infrequently(only accept connection).
So I picked simplicity rather than using `map[string]strcut{}` for time complexity.